cmd/config-host.go

```diff
@@ -48,26 +48,72 @@ func mainConfigHost(ctx *cli.Context) error {
 
 // hostMessage container for content message structure
 type hostMessage struct {
-	op        string
-	Status    string `json:"status"`
-	Alias     string `json:"alias"`
-	URL       string `json:"URL"`
-	AccessKey string `json:"accessKey,omitempty"`
-	SecretKey string `json:"secretKey,omitempty"`
-	API       string `json:"api,omitempty"`
+	op          string
+	prettyPrint bool
+	Status      string `json:"status"`
+	Alias       string `json:"alias"`
+	URL         string `json:"URL"`
+	AccessKey   string `json:"accessKey,omitempty"`
+	SecretKey   string `json:"secretKey,omitempty"`
+	API         string `json:"api,omitempty"`
 }
 
-// String colorized host message
+// Print the config information of one alias, when prettyPrint flag
+// is activated, fields contents are cut and '...' will be added to
+// show a pretty table of all aliases configurations
 func (h hostMessage) String() string {
 	switch h.op {
 	case "list":
-		message := console.Colorize("Alias", fmt.Sprintf("%s: ", h.Alias))
-		message += console.Colorize("URL", fmt.Sprintf("%-30.30s", h.URL))
-		if h.AccessKey != "" || h.SecretKey != "" {
-			message += console.Colorize("AccessKey", fmt.Sprintf("  %-20.20s", h.AccessKey))
-			message += console.Colorize("SecretKey", fmt.Sprintf("  %-40.40s", h.SecretKey))
-			message += console.Colorize("API", fmt.Sprintf("  %.20s", h.API))
+		dots := "..."
+
+		type Field struct {
+			colorTheme string
+			content    string
+			maxLen     int
+		}
+
+		fields := []Field{}
+
+		if h.prettyPrint {
+			// Add fields with pretty configuration
+			fields = append(fields,
+				Field{"Alias", h.Alias, -1},
+				Field{"URL", h.URL, 30},
+				Field{"AccessKey", h.AccessKey, 20},
+				Field{"SecretKey", h.SecretKey, 40},
+				Field{"API", h.API, 5},
+			)
+		} else {
+			fields = append(fields,
+				Field{"Alias", h.Alias, -1},
+				Field{"URL", h.URL, -1},
+				Field{"AccessKey", h.AccessKey, -1},
+				Field{"SecretKey", h.SecretKey, -1},
+				Field{"API", h.API, -1},
+			)
 		}
+
+		// Format fields and construct message
+		message := ""
+		for _, f := range fields {
+			// Skip empty field
+			// if len(f.content) == 0 {
+			//	continue
+			// }
+			// Default field format without pretty effect
+			fieldFormat := "%s  "
```

Comment:
It would be nice to convert this to a proper function perhaps? generalize it and add a unit test. ?